Makefile

```diff
@@ -1,9 +1,19 @@
+TARGETS           = darwin/amd64 linux/amd64 windows/amd64
```

Comment:
Looks you are using cross-compile on Linux/docker container for Windows binary. LGTM.